layouts/partials/download-list.hbs

```diff
@@ -1,5 +1,6 @@
 <section>
   <ul>
+    <li><a href="/{{site.locale}}/{{site.download.install-on-wsl.link}}/">{{site.download.install-on-wsl.text}}</a></li>
```

Comment:
```suggestion
    <li><a href="https://docs.microsoft.com/windows/dev-environment/javascript/nodejs-on-wsl">{{site.download.install-on-wsl.text}}</a></li>
```

You don't need to use the `site.locale` for the URL since it is an external link.

I also think this should likely be at the bottom of the list (I believe we have tended to add things in chronological order)